src/DatumConverter.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace r;
+
+use ReflectionFunction;
+use r\Query;
+use r\Datum\ArrayDatum;
+use r\Datum\BoolDatum;
+use r\Datum\NullDatum;
+use r\Datum\NumberDatum;
+use r\Datum\ObjectDatum;
+use r\Datum\StringDatum;
+use r\Exceptions\RqlDriverError;
+use r\FunctionQuery\RFunction;
+use r\Queries\Dates\Iso8601;
+use r\ValuedQuery\MakeArray;
+use r\ValuedQuery\MakeObject;
+use r\ValuedQuery\RVar;
+
+class DatumConverter
```

Comment:
DatumConverter doesn't have any state, so it's somewhat annoying and unnecessary having to do `$x = new DatumConverter; $x->...` every time we use one of its methods. I suggest making its methods static to avoid that.